app/views/hyrax/collections/show.html.erb

```diff
@@ -0,0 +1,141 @@
+<%# OVERRIDE Hyrax 5.0.1 and Hyku 6.0 to update the layout of the collection show page to be more visually appealing %>
```

Comment:
We may be able to apply this file back into Hyku.  But that is a matter of UI verification.